bb-core-tests/src/test/java/com/cognifide/qa/bb/test/loadable/TestPageObject.java

```diff
@@ -1,29 +1,27 @@
 /*
  * Copyright 2016 Cognifide Ltd..
  *
- * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
This change looks strange. Is this license formatted as in other files?
It should be..
(should have)